aws_lambda_powertools/utilities/parser/models/vpc_latticev2.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+from typing import Dict, Optional, Type, Union
+
+from pydantic import BaseModel, Field, validator
+
+
+class VpcLatticeV2RequestContextIdentity(BaseModel):
+    source_vpc_arn: Optional[str] = Field(None, alias="sourceVpcArn")
+    get_type: Optional[str] = Field(None, alias="type")
+    principal: Optional[str] = Field(None, alias="principal")
+    principal_org_id: Optional[str] = Field(None, alias="principalOrgID")
+    session_name: Optional[str] = Field(None, alias="sessionName")
+    x509_subject_cn: Optional[str] = Field(None, alias="X509SubjectCn")
+    x509_issuer_ou: Optional[str] = Field(None, alias="X509IssuerOu")
+    x509_san_dns: Optional[str] = Field(None, alias="x509SanDns")
+    x509_san_uri: Optional[str] = Field(None, alias="X509SanUri")
+    x509_san_name_cn: Optional[str] = Field(None, alias="X509SanNameCn")
+
+
+class VpcLatticeV2RequestContext(BaseModel):
+    service_network_arn: str = Field(alias="serviceNetworkArn")
+    service_arn: str = Field(alias="serviceArn")
+    target_group_arn: str = Field(alias="targetGroupArn")
+    identity: VpcLatticeV2RequestContextIdentity
+    region: str
+    time_epoch: float = Field(alias="timeEpoch")
+    time_epoch_as_datetime: datetime = Field(alias="timeEpoch")
+
+    @validator("time_epoch_as_datetime", pre=True, allow_reuse=True)
```

Comment:
Add a comment as to why this is needed so we don't forget.

I think there's something wrong here as the validator name doesn't match the logic - leftover perhaps?

Reply:
Weren't you gonna add a new property that returns datetime?

Reply:
I changed the name of this validator and added a comment.